src/main/java/frc/robot/commands/manipulator/pivot/PivotAdjust.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands.manipulator.pivot;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.Constants.ManipulatorConstants;
+
+public class PivotAdjust extends Command {
+  /** Creates a new TiltPIDCommand. */
+  private double distanceFromAprilTag;
+  private double setpoint;
+
+  public PivotAdjust() {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(RobotContainer.pivotSubsystem);
+    addRequirements(RobotContainer.limeSubsystem);
+    distanceFromAprilTag = RobotContainer.limeSubsystem.estimateDistance();
+    setpoint = RobotContainer.limeSubsystem.interpolateAngle(distanceFromAprilTag);
```

Comment:
these are only moving the arm at the start, if the robot moves, the angle will not change, maybe have these in the `execute()` to make sure that it updates constantly while active